recipes/pypy/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "pypy2.7" %}
```

Comment:
Why is the name `pypy2.7` instead of `pypy`?

Reply:
https://pypy.org/download.html  This is their naming scheme, so that it indicates CPython comparability.

Reply:
Right they have their own versioning in addition to CPython compat versioning. We might consider a `pypy` metapackage to make things a little easier.